contrib/actions/generate-actions.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+ACTIONS_HEAD = """
+name: Build Gluon
+on:
+  push:
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
+"""
+
+ACTIONS_TARGET="""
+  {target_name}:
+    name: {target_name}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Build
+        run: bash contrib/actions/run-build.sh {target_name}
+      - name: Archive build output
+        uses: actions/upload-artifact@v1
+        with:
+          name: output
+          path: output
+"""
+
+TARGETS= [
```

Comment:
Can we use `make list-targets` instead?

Reply:
Yes. Altered so the script accepts the input from `make list-targets`